crates/hdk/README.md

```diff
@@ -219,15 +193,15 @@ Holochain exposes a list of holochain-specific things that a wasm can do.
 For detailed documentation of the full list, see the `core/ribosome` module
 inside core, but some illustrative examples include:
 
-- `emit_signal`: publish data to subscribed clients
-- `encrypt` & `decrypt`: use the agent's keypair to encrypt and decrypt data
```

Comment:
these don't exist in this form anymore...

Reply:
`emit_signal` still does: https://docs.rs/hdk/0.0.100/hdk/p2p/fn.emit_signal.html

Reply:
oops, meant for only encrypt decrypt to come out

Reply:
Oh actually it's still there, it's just down below this comment block. Tweaked that text.